bin/faucet/Cargo.toml

```diff
@@ -58,6 +58,7 @@ reqwest          = { default-features = false, features = ["json"], version = "0
 serde_json       = { workspace = true }
 tokio            = { features = ["macros", "process"], workspace = true }
 tonic-web        = { version = "0.14" }
+uuid             = { features = ["v4"], version = "1.10" }
```

Comment:
as this import is now used by two crates, let's make it a `workspace` dependency